README.rst

```diff
@@ -45,7 +45,7 @@ With a few steps, you can start in your client repository using CI confiurations
   script:
     - .ci_config/travis.sh
 
-* Note that `.ci_config` is the REQUIRED name of the cloned folder; it is hardcoded on `industrial_ci` side so you must adhere to it.
+* Note: The name `.ci_config` is NO longer REQUIRED for the cloned folder since version 0.3.2; you can name it whatever you like.
```

Comment:
I see. Then why not making the folder hidden is a requirement because I don't see any benefit in client repositories to build industrial_ci? I've updated the commit so.